packages/@idemsInternational/env-replace/src/index.ts

```diff
@@ -195,8 +195,9 @@ class EnvReplaceClass {
   private logSummary() {
     const tableSummary = [];
     for (const [fileName, replacements] of Object.entries(this.summary)) {
-      for (const [varName, value] of Object.entries(replacements)) {
-        tableSummary.push({ fileName, varName, value });
+      for (let [varName, value] of Object.entries(replacements)) {
+        const truncatedValue = value.length > 40 ? value.slice(0, 37) + "..." : value;
```

Comment:
+1 - nice idea, yeah console tables don't wrap or flow nicely at all. 

nit(non-blocking): the `let` above can go back to `const` (guessing you had originally updated the value directly and then decided to use intermediate variable which I agree is a little bit cleaner)